src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js

```diff
@@ -171,6 +171,9 @@
             vm.propertiesToggled = true;
             vm.sideBarTab = "properties";
             $('#slider').css('display','block'); 
+            $timeout(function() {
+                 $scope.$broadcast('reCalcViewDimensions');
```

Comment:
can we use angular service instead of broadcast

Reply:
reCalcViewDimensions function is of rz-slider , so we have to broadcast this method when slider become visible.

Reply:
@johibkhan2  and @piyushmani  any update on this PR? Can this be merged?